sasmodels/models/_spherepy.py

```diff
@@ -79,23 +79,29 @@ def radius_effective(mode, radius):
     """Calculate R_eff for sphere"""
     return radius if mode else 0.
 
-def Iq(q, sld, sld_solvent, radius):
-    """Calculate I(q) for sphere"""
+# Variants for testing purposes: toggle True/False
+vectorized = True  # Whether to call a q vector or loop over q scalars.
```

Comment:
If they were for prototyping, consider removing them, if they're for testing, they and their dependents should be settable after loading the file.

Reply:
prototyping. It could be moved to the explore directory.